thematic/water-quality.thematic.mdx

```diff
@@ -10,11 +10,11 @@ media:
     url: https://unsplash.com/photos/4wFqHZ1ONnM
 
 about:
-  title: Ocean/Water Quality
-  description: "A combination of ground- and satellite-based tools provides a regional to global understanding of the impacts of water quality"
+  title: Observing Water Quality
+  description: "A combination of ground- and satellite-based tools provides a regional to global understanding of marine life and water quality"
 ---
 <Block>
 <Prose>
-TODO: Add content for the thematic are about page
+During the heat of the summer, the importance of monitoring and tracking algal blooms becomes readily apparent.  Those flocking to the cool waters of local lakes, such as [those in upstate New York](https://www.wamc.org/news/2022-08-19/new-york-communities-grapple-with-harmful-algal-blooms),  increasingly encounter  the menacing, shimmering green of algal blooms covering the surface of the water. Beyond the contrast of a slimy green surface with an expected sparkling blue one, algal blooms pose real risks to both humans and wildlife in terms of toxins and the depletion of dissolved oxygen in the water. Satellite observations of algal blooms are essential to maintaining a healthy environment.
```

Comment:
This description sounds very tied to the Algal bloom discovery, instead of being about a water quality thematic area. Is this intended?

Reply:
Yes and no. It's what Dee-Dee provided so I want to put it up on the site for her to see it. Once it's up there I am hoping she will realize how the content should probably be different than what she is currently writing.

Reply:
Ok. clear!